ds2799/model-ds2799-desc-compcop_smdl.json

```
@@ -0,0 +1,29 @@
+{
+  "Name": "ds2799_compcor",
+  "Description": "scrubbing compcor, see https://nilearn.github.io/stable/modules/generated/nilearn.interfaces.fmriprep.load_confounds_strategy.html",
+  "BIDSModelVersion": "1.0.0",
+  "Input": {
+    "task": "rest",
+    "space": "MNI152NLin2009cAsym"
+  },
+  "Nodes": [
+    {
+      "Level": "run",
+      "Name": "run",
+      "GroupBy": [
+        "run"
+      ],
```

"GroupBy" means the unique combinations of these variables form groups, so grouping only by "run" means all subjects/sessions that share the run index will be lumped together.

```suggestion
      "GroupBy": ["run", "session", "subject"],
```

The current semantics of PyBIDS allows "run" and "session" to be used as grouping variables even if absent from the dataset. I think we should make this explicit in the spec.

---

+1